tests/codeception-setup.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
```

Comment:
@yvesh @jatitoam @nibra, could this file be renamed to travisci-php-setup.sh (or similar) instead of the current codeception-setup? I think it will be more clear to understand what is this file for. 

Thougts?